app/swagger/swagger/v1/requests/post911_gi_bill_statuses.rb

```diff
@@ -83,7 +83,7 @@ class Swagger::V1::Requests::Post911GIBillStatuses
     property :end_date, type: %i[string null], example: '2012-12-01T05:00:00.000+00:00'
     property :facility_code, type: %i[string null], example: '12345678'
     property :facility_name, type: %i[string null], example: 'Purdue University'
-    property :participant_id, type: %i[string null], example: '11170323'
+    property :participant_id, type: %i[integer null], example: '11170323'
```

Comment:
Suggest removing quotes from the example now that this is an integer

Reply:
👌 my editor kept formatting the number strangely (no idea why), so I changed the example to `1234` but I *did* remove the quotes.  Thanks!